### ISSUE TITLE
Client incompatible with latest version of PyJWT (2.0.0)

### ISSUE THREAD

**Issue by dsmits** [vantage6/vantage6#10]
I would say the solution is to either freeze the version of PyJWT or add the required argument to jwt.decode

Exception:
```python
---------------------------------------------------------------------------
DecodeError                               Traceback (most recent call last)
<ipython-input-2-b5a5ca0518b4> in <module>
     11 
     12 client = vtgclient.Client(HOST, PORT)
---> 13 client.authenticate(USERNAME, PASSWORD)
     14 
     15 client.setup_encryption(None)

/opt/conda/lib/python3.8/site-packages/vantage6/client/__init__.py in authenticate(self, username, password)
    455         # belongs. This is usefull for some client side checks
    456         type_ = "user"
--> 457         id_ = jwt.decode(self.token, verify=False)['identity']
    458         user = self.request(f"user/{id_}")
    459         name = user.get("firstname")

/opt/conda/lib/python3.8/site-packages/jwt/api_jwt.py in decode(self, jwt, key, algorithms, options, **kwargs)
    111         **kwargs,
    112     ) -> Dict[str, Any]:
--> 113         decoded = self.decode_complete(jwt, key, algorithms, options, **kwargs)
    114         return decoded["payload"]
    115 

/opt/conda/lib/python3.8/site-packages/jwt/api_jwt.py in decode_complete(self, jwt, key, algorithms, options, **kwargs)
     77 
     78         if options["verify_signature"] and not algorithms:
---> 79             raise DecodeError(
     80                 'It is required that you pass in a value for the "algorithms" argument when calling decode().'
     81             )

DecodeError: It is required that you pass in a value for the "algorithms" argument when calling decode().

```

**Comment by bartvanb** [comment]
The above comment by Thomas appears to be still [up to date](https://github.com/jpadilla/pyjwt/blob/master/CHANGELOG.rst#dropped-deprecated-verify-param-in-jwtdecode) so I'm implementing that.

**Comment by frankcorneliusmartin** [comment]
This issue is still present in the client.

**Comment by frankcorneliusmartin** [comment]
Solved in #273
